src/CarbonAware.Aggregators/test/CarbonAwareAggregatorTests.cs

```diff
@@ -33,6 +33,74 @@ public void Setup()
         this.Aggregator = new CarbonAwareAggregator(this.Logger.Object, this.CarbonIntensityDataSource.Object);
     }
 
+    [TestCase("westus", "2021-11-17", "2021-11-20", ExpectedResult = 25)]
+    [TestCase("eastus", "2021-11-17", "2021-12-20", ExpectedResult = 60)]
+    [TestCase("westus", "2021-11-17", "2021-11-18", ExpectedResult = 20)]
+    [TestCase("eastus", "2021-11-19", "2021-12-30", ExpectedResult = 0)]
+    [TestCase("fakelocation", "2021-11-18", "2021-12-30", ExpectedResult = 0)]
+    public async Task<double> Test_Emissions_Average_FakeData(string location, string startTime, string endTime)
```

Comment:
These tests and corresponding code were removed.  Not sure how/why they were added back in, but might be worth thinking about what git strategy you used and investigate if there are others that avoid this type of issue.

Reply:
is likely the result of a 'bad merge' from msf:dev -- since the code removal appears to have just occurred...

Reply:
What happened here is plain ol' human error. I saw the merge, and thought that we'd accidentally deleted something, so I restored it. I'll get rid of these.